vagrant/private_network/Vagrantfile

```diff
@@ -12,6 +12,7 @@ Vagrant.configure(2) do |config|
     "host2" => "192.168.33.11",
     "host3" => "192.168.33.12",
     "host4" => "192.168.33.13",
+    "host5" => "192.168.33.14",
```

Comment:
That is the minimum to run the topology tests locally. I was thinking about creating another vagrant folder for these tests but maybe that could be another ticket?